app/helpers/compute_resources_helper.rb

```diff
@@ -38,6 +38,33 @@ def vm_pause_action(vm, authorizer = nil)
     display_link_if_authorized pause_action, opts, html.merge(:method => :put)
   end
 
+  def vms_table_attrs
+    if @compute_resource.supports_vms_pagination?
+      "data-table=server data-source=#{compute_resource_vms_path}"
+    else
+      "data-table=inline"
+    end
+  end
+
+  # we don't really know how many vms there are, but need to fool pagination to work
+  def vms_total_records
+    params['iDisplayStart'].to_i + 1 + [@vms.length, params['iDisplayLength'].to_i].min
+  end
```

Comment:
Either move it to ovirt controller or keep it here but rename it so it's obvious it is "faking" the page number. Like `vms_total_records_when_unknown`. Also the help text needs more love :-)